### PR TITLE
#2516924 - Create Membership Renewal Activity

### PR DESCRIPTION
**Pull request by colemanw**

### CODE REVIEW

File: includes/wf_crm_webform_postprocess.inc [hunk 1]
```diff
@@ -1102,6 +1102,22 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       $params['skipStatusCal'] = !empty($params['status_id']);
 
       $result = wf_civicrm_api('membership', 'create', $params);
+      
+      //Issue #2516924 If existing membership create renewal activity
+      foreach ($result['values'] as $memId => $membership) {
+        if ($params['membership_type_id'] == $memId) {
```
Comment (colemanw) [inline includes/wf_crm_webform_postprocess.inc:1108]: It looks like you're comparing apples to oranges here. Why would it matter if the membership type matched the membership id?

**Comment by colemanw** [comment]
@relldoesphp can you take a look?

**Comment by relldoesphp** [comment]
Sorry I meant to check the id not the membership type id, it was to check if the membership id created matched the membership id passed in

**Comment by colemanw** [comment]
Ok. Actually I don't think we really need to `foreach` the results of the api call since there can only possibly be 1 record (or 0 if something went wrong). Just need to check `if (!empty($result['id']) && !empty($params['id']))`

**Comment by colemanw** [comment]
Ok fixed with 3222a31

**Pull request merged**